index.js

```diff
@@ -218,6 +218,7 @@ class AntiSpam extends EventEmitter {
 			}
 
 			try {
+				const tempMember = message.member
```

Comment:
Add missing semicolon 😅 

```suggestion
				const tempMember = message.member;
```

Reply:
Wow